src/Data/Skills/sup_int.lua

```diff
@@ -5099,7 +5112,20 @@ skills["SupportUnleashPlus"] = {
 	excludeSkillTypes = { SkillType.Totem, SkillType.Trap, SkillType.Mine, SkillType.Triggered, SkillType.ManaCostReserved, SkillType.Vaal, SkillType.Instant, SkillType.Channelled, },
 	plusVersionOf = "SupportHandcastAnticipation",
 	statDescriptionScope = "gem_stat_descriptions",
+	statMap = {
+		["support_anticipation_rapid_fire_count"] = {
+			mod("SealCount", "BASE", nil),
+		},
+		["support_anticipation_charge_gain_interval_ms"] = {
+			mod("SealGainFrequency", "BASE", nil),
+			div = 1000,
+		},
+		["support_spell_rapid_fire_repeat_use_damage_+%_final"] = {
+			mod("SealRepeatPenalty", "MORE", nil),
+		},
+	},
 	baseMods = {
+		flag("HasSeals"),
```

Comment:
@scristall FYI, you didn't have this in the `.txt` file, so it was removed when I was doing the export.  In the future use `#baseMod flag("HasSeals")` in the .txt to add it